packages/calcite-components/src/components/slider/slider.tsx

```diff
@@ -282,19 +282,19 @@ export class Slider
   /** Specifies the interval to move with the page up, or page down keys. */
   @property({ reflect: true }) pageStep: number;
 
-  /** When `true`, sets a finer point for handles. */
+  /** When present, sets a finer point for handles. */
   @property({ reflect: true }) precise = false;
 
   /**
-   * When `true` and the component resides in a form,
+   * When present, and the component resides in a form,
```

Comment:
I think we may want to not add this first comma to keep consistency with other "resides in form" docs.


```suggestion
   * When present and the component resides in a form,
```